### ISSUE TITLE
[BUG] TypeError: Can't instantiate abstract class MPS_Accelerator with abstract methods

### ISSUE THREAD

**Issue by katopz** [microsoft/DeepSpeed#5132]
**Describe the bug**
TypeError: Can't instantiate abstract class MPS_Accelerator with abstract methods handles_memory_backpressure, resolves_data_dependency, use_host_timers

**To Reproduce**
1. git clone https://github.com/OpenBMB/MiniCPM.git
2. follow setup step.
3. run `!bash lora_finetune.sh` via `lora_finetune.ipynb`

**Expected behavior**
runnable

**ds_report output**
Please run `ds_report` to give us details about your setup.

**Screenshots**
```
    class CudaEventTimer(object):
  File "/Users/katopz/git/MiniCPM/finetune/.venv/lib/python3.11/site-packages/deepspeed/utils/timer.py", line 33, in CudaEventTimer
    def __init__(self, start_event: get_accelerator().Event, end_event: get_accelerator().Event):
                                    ^^^^^^^^^^^^^^^^^
  File "/Users/katopz/git/MiniCPM/finetune/.venv/lib/python3.11/site-packages/deepspeed/accelerator/real_accelerator.py", line 184, in get_accelerator
    ds_accelerator = MPS_Accelerator()
                     ^^^^^^^^^^^^^^^^^
TypeError: Can't instantiate abstract class MPS_Accelerator with abstract methods handles_memory_backpressure, resolves_data_dependency, use_host_timers
```

**System info (please complete the following information):**
 - OS: macOS 14.2.1 (23C71)
 - metal
 - Python 3.11.7

**Launcher context**
deepspeed

**Docker context**
no

**Additional context**

**Comment by mrwyattii** [comment]
Thank you for reporting this issue. It should be resolved in #5134. Could you test with this branch and verify? `pip install git+github.com/microsoft/deepspeed@mrwyattii/fix-mps-accel`

**Comment by ziyigogogo** [comment]
> Thank you for reporting this issue. It should be resolved in #5134. Could you test with this branch and verify? `pip install git+github.com/microsoft/deepspeed@mrwyattii/fix-mps-accel`

hi there , met the same issue now , but the cmd you provided here is not working anymore.
showed that : Hint: It looks like a path. File 'git+github.com/microsoft/deepspeed@mrwyattii/fix-mps-accel' does not exist.

can you share a new path please?

**Comment by monachitnis** [comment]
> hi there , met the same issue now , but the cmd you provided here is not working anymore. showed that : Hint: It looks like a path. File 'git+github.com/microsoft/deepspeed@mrwyattii/fix-mps-accel' does not exist.
> 
> can you share a new path please?

https://github.com/microsoft/DeepSpeed/pull/5134 is merged, so you no longer need branch, and can use master, correct?
